framework/include/wifi_manager/wifi_manager.h

```diff
@@ -55,11 +55,26 @@ typedef enum {
 	SOFTAP_MODE
 } wifi_manager_mode_e;
 
+typedef enum {
+	WIFI_SCAN_FAIL = -1,
+	WIFI_SCAN_SUCCESS,
+} wifi_manager_scan_result_e;
+
+struct wifi_manager_scan_info_s {
+	char ssid[33];	// 802.11 spec defined unspecified or uint8
+	char bssid[18];	// char string e.g. xx:xx:xx:xx:xx:xx
+	int8_t rssi;		// received signal strength indication
+	struct wifi_manager_scan_info_s *next;
+};
+
+typedef struct wifi_manager_scan_info_s wifi_manager_scan_info_s;
+
 typedef struct {
-	void (*sta_connected)(void);
-	void (*sta_disconnected)(void);
-	void (*softap_sta_join)(void);
-	void (*softap_sta_leave)(void);
+	void (*sta_connected)(void);		// in station mode, connected to ap
+	void (*sta_disconnected)(void);		// in station mode, disconnected from ap
+	void (*softap_sta_joined)(void);	// in softap mode, a station joined
+	void (*softap_sta_left)(void);		// in softap mode, a station left
+	void (*sta_scan_ap_done)(wifi_manager_scan_info_s **, wifi_manager_scan_result_e); // in station mode, scanning ap is done
```

Comment:
Modify the function name(sta_scan_ap_done) and comment. Scanning is available both in soft ap mode and in sta mode.

Reply:
I rename sta_scan_ap_done with scan_ap_done and modify related comments.